src/frequenz/sdk/microgrid/_power_distributing/_component_managers/_battery_manager.py

```diff
@@ -652,6 +665,12 @@ async def _set_distributed_power(
                 api.set_power(inverter_id, power.as_watts())
             )
             for inverter_id, power in distribution.distribution.items()
+            # Don't repeat setting zero power to inverters, to not interfere
+            # with any lower-level users of the inverters.
+            if not (
+                power == Power.zero()
+                and self._last_set_power.get(inverter_id) == Power.zero()
+            )
         }
```

Comment:
I thought actors set lower and upper power limit, to not interfere with lower-level actors. 
And if actor won't use battery, then it can set None (then lower level actor will have full control).